gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -90,6 +96,18 @@ export default function NxFormLayoutExample() {
           Blue
         </NxRadio>
       </fieldset>
+      <div className="nx-form-group">
+        <label className="nx-label">
+          <span className="nx-label__text">Select</span>
+          <select className="nx-form-select" value={selectVal} onChange={onSelectChange}>
+            <option>Option 1</option>
```

Comment:
The typical way to use `<option>` elements is to give each one a `value` that is more of the machine-readable value whereas the child text is the human readable form.

Reply:
> The typical way to use `<option>` elements is to give each one a `value` that is more of the machine-readable value whereas the child text is the human readable form.

The task was just to style the `<select>` and I would hope that any developer can figure that out themselves without us having to create an example

Reply:
> The typical way to use `<option>` elements is to give each one a `value` that is more of the machine-readable value whereas the child text is the human readable form.

https://github.com/sonatype/sonatype-react-shared-components/pull/203/commits/a329a17b7cda35bbf29e06fca284f1b3ae6255ea